float8_experimental/float8_dynamic_linear.py

```diff
@@ -62,8 +62,12 @@ class Float8DynamicLinear(torch.nn.Linear):
     def __init__(self, **super_kwargs):
         super().__init__(**super_kwargs)
 
+        self.activation_scale: Optional[torch.Tensor] = None
```

Comment:
we'd probably need serialization for this, either by making it a buffer or adding custom state dict save/load